### PR TITLE
Compatibility with FontAwesome 5 (upgrade)

### PR DESCRIPTION
**Pull request by jeojhx**
https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4

### CODE REVIEW

**Comment by fisharebest** [comment]
Until this PR is merged, a simple workaround is to add the font-awesome category to the icon name.

e.g. for `fas fa-university`, you specify your icon as `university fas`

File: leaflet-beautify-marker-icon.js [hunk 1]
```diff
@@ -148,8 +148,16 @@
                 if (options.spin) {
                     spinClass = ' fa-spin';
                 }
-
-                return '<i class="' + options.prefix + ' ' + options.prefix + '-' + options.icon + spinClass + '" style="' + innerIconStyle + '"></i>';
+                /*fontawesome 5.4.1
+                https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4
+                
+                New Prefix	|  Icon Style        	 |  SVG + JS Filename	| Web Font Filename	| Availability
+                fab	        |  Font Awesome Brands | brands.js	        |  fa-brands-400.*  |	Free
```
Comment (Beck888) [inline leaflet-beautify-marker-icon.js:155]:
```suggestion
                fab	        |  Font Awesome Brands | brands.js	        |  fa-brands-400.*  |	Free
```

**Comment by jeojhx** [comment]
Compatibility with FontAwesome 5

**Comment by jeojhx** [comment]
Hi, @masajid390 can you review this pull request, it's valid yet, maybe can you merge.. grettings...

**Comment by masajid390** [comment]
You could override the prefix property of icon options. i.e. see `prefix: "fas"`
 
`var options = {
          icon: 'bus',
          borderColor: '#b3334f',
          textColor: '#b3334f',
          prefix: "fas"
        };
        L.marker([33.89093747081252, 72.9107666015625], {
          icon: L.BeautifyIcon.icon(options),
          draggable: true
        }).addTo(map).bindPopup("I'm Beautify");`

**Comment by jeojhx** [comment]
Soryy @masajid390 but this is not possible because you use the prefix on the begin of icon name,

>` '<i class="' + options.prefix + ' ' + options.prefix + '-' + options.icon + spinClass + '" style="' + innerIconStyle + '"></i>'`

; in your example this render 

> `'<i class="'fas fas-bus" ></i>' 
`
when really, i want

> ` '<i class="'fas bus" ></i>' `

:)

**Comment by masajid390** [comment]
In that case, you could set `prefix` as empty, and set icon like ` fas bus`.

`var options = { icon: ' fas bus', borderColor: '#b3334f', textColor: '#b3334f', prefix: "" }; L.marker([33.89093747081252, 72.9107666015625], { icon: L.BeautifyIcon.icon(options), draggable: true }).addTo(map).bindPopup("I'm Beautify");`

I hope that would work :-)

**Pull request closed without merging**